decision_maker.go

```diff
@@ -219,13 +219,17 @@ func diffRelease(r *release) string {
 	exitCode := 0
 	msg := ""
 	colorFlag := ""
+	suppressDiffSecretsFlag := ""
 	if noColors {
 		colorFlag = "--no-color "
 	}
+	if suppressDiffSecrets {
+		suppressDiffSecretsFlag = "--suppress-secrets "
+	}
 
 	cmd := command{
 		Cmd:         "bash",
-		Args:        []string{"-c", "helm diff " + colorFlag + "upgrade " + r.Name + " " + r.Chart + getValuesFiles(r) + " --version " + strconv.Quote(r.Version) + " " + getSetValues(r) + getSetStringValues(r) + getDesiredTillerNamespaceFlag(r) + getTLSFlags(r) + getTimeout(r) + getNoHooks(r)},
+		Args:        []string{"-c", "helm diff " + colorFlag + suppressDiffSecretsFlag + "upgrade " + r.Name + " " + r.Chart + getValuesFiles(r) + " --version " + strconv.Quote(r.Version) + " " + getSetValues(r) + getSetStringValues(r) + getDesiredTillerNamespaceFlag(r) + getTLSFlags(r) + getTimeout(r) + getNoHooks(r)},
```

Comment:
thanks @hatemosphere this is good. 
Can you please resolve the merge conflict by removing `getTimeout(r)` as per #131 
Also `getNoHooks(r)` need to be removed here (fixing another issue :-) as helm diff does not have a `no-hooks` option.